### PR TITLE
Add changelog to the extra-source files.

### PR DESCRIPTION
**Pull request by qnikst**
This fix will create a changelog item on hackage.

[Review on Reviewable](https://reviewable.io/reviews/tweag/network-transport-zeromq/32)

### CODE REVIEW

**Comment by qnikst** [comment]
@facundominguez please review.

File: ChangeLog.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+0.2.1
+-----
+  * Verify that exceptions doesn't leak to the outter scope
```
Comment (facundominguez) [inline ChangeLog.md]: Verify that exceptions **don't** leak to the **outer** scope

**Comment by qnikst** [comment]
Fixed

**Comment by facundominguez** [comment]
:+1:

**Pull request merged**